src/main/java/gov/nasa/pds/web/ui/utils/TabularData.java

```diff
@@ -74,12 +74,13 @@ public TabularData(URL fileUrl, List<ColumnInfo> columnInfos, long startByte,
         final BufferedReader br = new BufferedReader(new InputStreamReader(fileUrl.openStream()));
         String currLine = null;
         int lineNumber = 1;
-
+        long llineNumber = 1;
```

Comment:
Can we delete `int lineNumber = 1` and all the increments of it later on `lineNumber++`? It looks like `llineNumber` replaces it? 🤷

Reply:
@al-niessner ☝️

Reply:
Primitives are so annoying. No, cannot delete the int. If we go with long, then have to convert long to int for the call a couple of lines down within the loop that has an int in its constructor. So, either two variable (bleh) or cast between in int/long in a loop. Personally I would go with the latter but the static analysis seems to dislike that idea. Changing the constructor would require significant code changes as it used lots of places that would then cause more change then more change as we go from int to long without casting because everything is in a loop in validate.